apps/charterafrica/src/payload/blocks/BlogContent.js

```diff
@@ -0,0 +1,14 @@
+import blockFields from "../fields/blockFields";
+import richText from "../fields/richText";
+
+const BlogContent = {
+  slug: "blogContent",
```

Comment:
I wouldn't call this BlogContent since we'll use it Reseach, privacy policy page, etc. etc. May be RichText, etc.